### PR TITLE
Plotting code update, HEM veto selection is applied in darkhiggs.py, changes on signal processes cross section numbers 

### PR DESCRIPTION
**Pull request by Quantumapple**
1. New definitions for HEM veto: "isHEMelectron", "isHEMmuon", "isHEMjet"

2. Added new selections when HEM veto is applied

3. Change the cross section number for signal MC samples.

4. Now plots include data/MC ratio

### CODE REVIEW

File: grinder/analysis/darkhiggs.py [hunk 1]
```diff
@@ -461,13 +511,18 @@ def process(self, df):
                 #baggy = (fj_nclean>0)&(fj_clean.pt.max()>160)&(abs(u[k].delta_phi(j_clean)).min()>0.8)&(u[k].pt>250)
                 #skinny = (j_nclean>0) & (j_clean.pt.max()>100) & (abs(u[k].delta_phi(j_clean)).min()>0.5) & (u[k].pt>250)
                 #skinny_no_baggy = ~baggy&skinny
-                selections.add(k+'baggy', (fj_nclean>0)&(fj_clean.pt.max()>160)&(abs(u[k].delta_phi(j_clean)).min()>0.8)&(u[k].pt>250))
+                #selections.add(k+'baggy', (fj_nclean>0)&(fj_clean.pt.max()>160)&(abs(u[k].delta_phi(j_clean)).min()>0.8)&(u[k].pt>250))
+                selections.add(k+'baggy', (fj_nclean>0)&(fj_clean.pt.max()>160)&(u[k].pt>250))
```
Comment (areinsvo) [inline grinder/analysis/darkhiggs.py]: The rest of the PR looks good, but why did we drop the delta phi cut here? @mcremone @Quantumapple

Comment (Quantumapple) [reply grinder/analysis/darkhiggs.py]: I checked my note in August, at that time, we made yields tables when we applied "taggers". We found that yields in dilepton CRs are too small, so Matteo decided to remove delta phi cut and dilepton mass cut. Now we only recover dilepton mass cut. @areinsvo

Comment (areinsvo) [reply grinder/analysis/darkhiggs.py]: Can you please add this back in? Unless I'm mistaken, the low yields are caused by the missing cleaning requirements mentioned in https://github.com/LPC-DM/decaf/issues/37, and not by this delta_phi cut.

Comment (Quantumapple) [reply grinder/analysis/darkhiggs.py]: No problem!

**Pull request merged**